cnxeasybake/oven.py

```diff
@@ -269,6 +270,13 @@ def bake(self, element, last_step=None):
 
         # Do label/link updates
 
+        # Add an empty string to each element just to make sure the element is closed
```

Comment:
I might be happier with this if it were in the cli script, rather than the Oven. Or else, extend the comment to explicitly mention  making it parse in an HTML5 (non-XHTML) context.